js/apps/account-ui/src/resources/ResourcesTab.tsx

```diff
@@ -154,7 +154,9 @@ export const ResourcesTab = ({ isShared = false }: ResourcesTabProps) => {
             <Th aria-hidden="true" />
             <Th>{t("resourceName")}</Th>
             <Th>{t("application")}</Th>
-            <Th>{!isShared ? t("permissionRequests") : ""}</Th>
+            <Th aria-hidden={isShared ?? true}>
```

Comment:
Since `isShared` is always defined there should be no need to fall back to another value.